cmsdb/campaigns/run2_2017_nano_v9/mttbar.py

```diff
@@ -0,0 +1,1808 @@
+# coding: utf-8
+
+"""
+Signal samples for m(ttbar) line search
+"""
+
+import cmsdb.processes as procs
+from cmsdb.campaigns.run2_2017_nano_v9 import campaign_run2_2017_nano_v9 as cpn
+
+
+cpn.add_dataset(
+    name="zprime_tt_m_400_w_40_madgraph",
```

Comment:
Maybe just a minor comment here, but maybe skipping the underscore between parameter name (`m`) and value `400` could make this more readable

→ `zprime_tt_m400_w40_madgraph`